imap_processing/tests/external_test_data_config.py

```diff
@@ -183,11 +183,19 @@
     ("imap_ultra_l1c-90sensor-sc-pointing-phi-test_20250101_v000.csv", "ultra/data/l1/"),
     ("imap_ultra_l1c-90sensor-sc-pointing-index-test_20250101_v000.csv", "ultra/data/l1/"),
     ("imap_ultra_l1c-90sensor-sc-pointing-bsf-test_20250101_v000.csv", "ultra/data/l1/"),
+    ("imap_ultra_l1c-90sensor-sc-pointing-theta_20250101_v001.csv", "ultra/data/l1/"),
+    ("imap_ultra_l1c-90sensor-sc-pointing-phi_20250101_v001.csv", "ultra/data/l1/"),
+    ("imap_ultra_l1c-90sensor-sc-pointing-index_20250101_v001.csv", "ultra/data/l1/"),
+    ("imap_ultra_l1c-90sensor-sc-pointing-bsf_20250101_v001.csv", "ultra/data/l1/"),
```

Comment:
Above this there are `-test` files with these same names. Do you want to replace those or remove those now?

Reply:
Do the files still get downloaded by everyone because this is a generic external downloads? Or is each individual test requesting a single external download file so this doesn't impact everyone with the file sizes?

Reply:
I think you'd request the download function explicitly from within the test itself. Since you are skipping it automatically already, that should work and not need the external test mark even.